src/components/contracts/ExtendContractForm.vue

```diff
@@ -0,0 +1,360 @@
+<template>
+  <v-form>
+    <v-alert
+      v-if="firstShiftOfContract !== null"
+      data-cy="alert-step-one"
+      type="info"
+    >
+      {{ $t("contracts.disableDateChangeInfo") }}
+    </v-alert>
+    <v-alert
+      v-if="specificContractExpired(contract)"
```

Comment:
Wäre es möglich, beide `v-alerts` in einen Container zu packen? Denn es werden sowieso immer beide angezeigt, wenn `specificContractExpired`.

Vorschlag: einen Alert als `type="info"` (sieht vllt. nicht ganz so streng aus). Ist fast egal, ob der 1. oder 2.

Reply:
Ich finde zwei alerts mit wenig Text attraktiver zu lesen als eine mit viel Text.

Reply:
Ja, auf jeden Fall - ich habe mich missverständlich ausgedrückt. Ist schon richtig so!